content/post/personas.md

```diff
@@ -0,0 +1,49 @@
+| <img class="image" src="/images/persona3.svg" alt="Librarian persona" title="Librarian persona" style="vertical-align:top;"><br/>**Gabriele**<br/>*Librarian (Università della Svizzera italiana - with dedicated SONAR portal)* | **Shared portal**<ul><li>export metadata of publications according to various criteria (author, project, institution, document type, etc.) so that I can reuse the data (showcase, report, FNS, etc.)</li></ul>**Dedicated portal**<ul><li>upload PDF of pre-prints and post-prints (OA or not) of my institution's researchers so that they comply with the OA policy</li><li>indicate for each full-text the OA status (incl. embargo periods) so that they are publicly available or not.</li><li>enter metadata about publications (typology, rights, access, etc.) so that they are searchable</li><li>create profiles for researchers with specific rights</li><li>import metadata from other platforms (ORCID, Google Scholar, etc.) so that I don't have to enter it manually</li><li>validate full-text inputs of the researchers so that they complies with legal restrictions</li><li>validate and ev. complete metadata entered by researchers so that it complies with standards</li><li>export metadata of publications according to various criteria (author, project, institution, document type, …) so that I can reuse the data (showcase, report, etc.)</li><li>request or harvest publications according to various criteria (author, project, institution, document type, …) so that I can integrate them into other platforms (website, etc.)</li><li>ensure data integrity, readability and access so that we can access it in the long term</li></ul> |
+
+3 other personas, not detailed in this blog post, complete the scope of user needs:
+
+* Corinna, professor and researcher at University of Basel (without dedicated SONAR portal)
+* Gaston, information specialist at SNSF
+* Ruth, journalist at Neue Zürcher Zeitung
+
+### And now what?
+
+This work is still in progress and we are improving iteratively the personas. Based on this, we defined the functional specifications, with various detailed scenarios for each persona. We are currently working on the structure of the portal and conceiving the first mockups of the interface.
```

Comment:
conceiving the first mockups of the interface -> conceiving the mockups of the interface